staging/src/k8s.io/apimachinery/pkg/watch/mux_test.go

```diff
@@ -90,8 +92,8 @@ func TestBroadcaster(t *testing.T) {
 
 func TestBroadcasterWatcherClose(t *testing.T) {
 	m := NewBroadcaster(0, WaitIfChannelFull)
-	w := m.Watch()
-	w2 := m.Watch()
+	w, _ := m.Watch()
+	w2, _ := m.Watch()
```

Comment:
```suggestion
	w, err := m.Watch()
	if err != nil {
	  t.Fatalf(err)
	}
	w2, _ := m.Watch()
	if err != nil {
	  t.Fatalf(err)
	}
```

I would not swallow the errors

Reply:
Done